scripts/miniapps.py

```diff
@@ -168,28 +183,33 @@ def chol(
     suffix="na",
     extra_flags="",
     env="",
+    typ="d",
```

Comment:
Is `type` reserved for something as a name?

Reply:
https://www.w3schools.com/python/ref_func_type.asp

Reply:
:sob: 

`datatype`? Not insisting on this at all (as you can see from my approval).

Reply:
https://github.com/eth-cscs/DLA-Future/pull/1010/commits/4425298ac53c5e5684e15dfb2bc6432de1db4a10